packages/core/contracts-ovm/insured-bridge/external/AVM_L2CrossDomainEnabled.sol

```diff
@@ -0,0 +1,15 @@
+pragma solidity >=0.7.6;
+
+abstract contract AVM_L2CrossDomainEnabled {
```

Comment:
This file used to be AVM_CrossDomainEnabled but I split it into the L1 and L2 exclusive utility functions.